### ISSUE TITLE
Menu bar accelerators do not work when RichTextFX control has focus

### ISSUE THREAD

**Issue by JFormDesigner** [TomasMikula/RichTextFX#185]
When the a RichTextFX control is focused, the menu bar accelerators do not work.

In Markdown Writer FX I workaround this by installing all menu bar accelarators into the RichTextFX controls. See: [MarkdownEditorPane.java#L130-L132](https://github.com/JFormDesigner/markdown-writer-fx/blob/master/src/main/java/org/markdownwriterfx/editor/MarkdownEditorPane.java#L130-L132) and [MainWindow.java#L302-L327](https://github.com/JFormDesigner/markdown-writer-fx/blob/master/src/main/java/org/markdownwriterfx/MainWindow.java#L302-L327)

Made a little test app that demonstrates that menu bar accelarators do not work when a RichTextFX control has focus, but do work when a plain JavaFX text area has focus:
```java
import javafx.application.Application;
import javafx.scene.Scene;
import javafx.scene.control.*;
import javafx.scene.input.KeyCombination;
import javafx.scene.layout.BorderPane;
import javafx.stage.Stage;
import org.fxmisc.richtext.StyleClassedTextArea;

public class RichTextWithMenuBarTest extends Application {
	public static void main(String[] args) {
		launch(args);
	}

	@Override
	public void start(Stage primaryStage) {
		StyleClassedTextArea richTextArea = new StyleClassedTextArea();
		richTextArea.replaceText("RichTextFX control - Ctrl+O does not work when focused");

		TextArea textArea = new TextArea("TextArea control - Ctrl+O works when focused");

		BorderPane root = new BorderPane(richTextArea);
		root.setTop(createMenuBar());
		root.setBottom(textArea);

		Scene scene = new Scene(root, 600, 400);
		primaryStage.setScene(scene);
		primaryStage.setTitle("RichText With MenuBar Test");
		primaryStage.show();
	}

	private MenuBar createMenuBar() {
		MenuItem fileOpenItem = new MenuItem("Open");
		fileOpenItem.setAccelerator(KeyCombination.valueOf("Ctrl+O"));
		fileOpenItem.setOnAction(e -> {
			Alert alert = new Alert(Alert.AlertType.INFORMATION, "Ctrl+O pressed");
			alert.showAndWait();
		});

		Menu fileMenu = new Menu("File", null, fileOpenItem);
		return new MenuBar(fileMenu);
	}
}
```

Karl

**Comment by QAston** [comment]
Having the same problem here, Windows, JDK 1.8.0_45

**Comment by TomasMikula** [comment]
I tried your example, but I cannot reproduce the problem. The Alert dialog gets displayed regardless of which text area is focused. This is on Mac and JDK 8u60. Strangely enough, with JDK 8u40 I get the dialog displayed twice in a row, which might be the same issue as #184.

**Comment by JFormDesigner** [comment]
I've noticed the problem on Windows 8.1 and JDK 8u60.

**Comment by TomasMikula** [comment]
OK, I see an issue in the code.

**Comment by TomasMikula** [comment]
Please, try the newest snapshot.

**Comment by QAston** [comment]
The snapshot works, thanks for a very quick fix!

**Comment by JFormDesigner** [comment]
Thanks. Works now in Markdown Writer FX without workaround.

**Comment by TomasMikula** [comment]
I just published `0.6.10` with this fix.
